agent/php_execute.c

```diff
@@ -1013,7 +977,7 @@ typedef struct {
  */
 static void nr_php_execute_metadata_init(nr_php_execute_metadata_t* metadata,
                                          zend_op_array* op_array) {
-#ifdef PHP7
+#if ZEND_MODULE_API_NO >= ZEND_7_0_X_API_NO /* PHP7+ */
   if (op_array->scope && op_array->scope->name && op_array->scope->name->len) {
     metadata->scope = op_array->scope->name;
```

Comment:
Could we use `zend_copy_string()` to simplify code?

Reply:
Thanks for keeping an eye out for possible simplifications.

It's possible we could call zend_string_copy instead of zend_string_addref but it wouldn't simplify the code since all other lines would need to remain the same.  Also, changing the current implementation that uses addref and that affects all user functions that come through php_execute_enabled and the testing effort involved seems beyond the scope of this PR.

```
static zend_always_inline zend_string *zend_string_copy(zend_string *s)
{
        if (!ZSTR_IS_INTERNED(s)) {
                GC_ADDREF(s);
        }
        return s;
}
```

vs 

```
static zend_always_inline uint32_t zend_string_addref(zend_string *s)
{
        if (!ZSTR_IS_INTERNED(s)) {
                return GC_ADDREF(s);
        }
        return 1;
}
```